tools/classes.js

```diff
@@ -220,6 +220,7 @@ var classes = [
   'foam.dao.PurgeRecordCmd',
   'foam.dao.UniqueConstraintException',
   'foam.dao.FUIDAO',
+  'foam.dao.CompositeRelationship',
```

Comment:
Why do you have this, when there is nothing in CompositeRelationship for java? It doesn't even have class/type info even on the properties.

Reply:
Ya this was initially when I was thinking of it working like relationship and was thinking could use java impls later on. Can remove it.

Reply:
removed.